### ISSUE TITLE
Defer to watching only one npm package with `bin/hack --watch-js`

### ISSUE THREAD

**Issue by gazayas** [bullet-train-co/bullet_train-core#39]
## TL;DR
Changes to JS when watching multiple npm packages in parallel won't be reflected until the server is restarted.

Changes to JS when watching only one npm package <b><i>are</i></b> reflected without any issues.

## Details

With the current `bin/hack --watch-js` script, I was able to run the different yarn processes in parallel to watch the following npm packages:
1. @bullet-train/bullet-train
2. @bullet-train/fields
3. @bullet-train/bullet-train-sortable

When there are updates to JavaScript files, the watcher runs the `update-local-installs` command to build the changes, and the command would show up in the logs when we were watching multiple packages in parallel. The script for the `@bullet-train/bullet-train` package can be found [here](https://github.com/bullet-train-co/bullet_train-core/blob/dc84dca0feed3ce0dc56ee0da7006a4a90dc25c7/bullet_train/package.json#L24) and the logs look like this:
```
[update-local-installs] [nodemon] starting `npm run -s update-local-installs`
  [update-local-installs] Build "bulletTrain" to app/assets/javascripts:
        2.57 kB: bullet-train.js.gz
        2.24 kB: bullet-train.js.br
  [update-local-installs] @bullet-train/bullet-train@1.0.99 published in store.
  [update-local-installs] Pushing @bullet-train/bullet-train@1.0.99 in /Users/gabriel_zayas/work/bt/bullet_train
  [update-local-installs] Package @bullet-train/bullet-train@1.0.99 linked ==> /Users/gabriel_zayas/work/bt/bullet_train/node_modules/@bullet-train/bullet-train
  [update-local-installs] [nodemon] clean exit - waiting for changes before restart
```

However, the changes aren't reflected when watching multiple packages. I'm assuming there's something murky going on because we're running the commands in parallel, but I'm not 100% sure what the cause is.

## `bin/hack --watch-js (package_name)`
I'm personally ok with editing `bin/hack --watch-js` so it takes an argument and watches only one package at a time. This way we can be sure the changes are reflected.

I originally wrote it that way, and although I'd like to watch all the packages at once, the result probably isn't worth the effort currently. Also, developers probably don't need to watch for changes in <i>all</i> of the packages' JS files at one time, as long as their local changes in other packages are reflected. They would just have to restart the server if they choose to make changes to packages that aren't being watched.
